source/js/components/post-code-validation.js

```diff
@@ -0,0 +1,58 @@
+import { countriesAndPostCodes } from "./post-codes-list.js";
+
+// List of countries to use for reference
+const listOfCountries = countriesAndPostCodes;
+
+// The container for the post-code input field
+const postCodeInputContainer = document.querySelector(
+  ".post-code-input-container"
+);
+
+// The container that houses both the post code and city inputs
+const formAndCityContainer = document.querySelector(".form__group--city-post");
+
+// The select object that allows users to select their country
+const countrySelector = document.getElementById("id_country");
+
+// Calling the function once to check in case the user refreshes the page,
+// we want the input to remain one column if needed
+if (countrySelector) {
+  checkForCountrysPostCode();
```

Comment:
just for ease of reason, let's make this `checkForCountryPostCode()`, since `countrys` looks like an improper plural and may cause typo bugs in the future.

(and let's do that for the other instances too)